seeds/test/UserSeed.js

```diff
@@ -0,0 +1,8 @@
+module.exports = [
```

Comment:
I've tried to do it exactly that way, but strangely enough, it doesn't work. There are no errors, but it just doesn't load it. May be a problem with the used package seed-mongoose.